src/components/LikeMenuBtn/LikesMenuBtn.jsx

```diff
@@ -0,0 +1,15 @@
+import { useVideoContext, useAuthContext } from "../../context/context";
```

Comment:
suggestion : you dont need to make this component as you have video card component, in video card you have dot options, there you can add to add to liked btn and remove from like btn basically you can toggle there, and as per prd you need to like/dislike btn functionality in home/video-listing page, so i would recommend you should do like i suggested.

Reply:
![image](https://user-images.githubusercontent.com/68814457/160917530-652c636b-d5c2-4643-bd7c-4a60d790732e.png)

Reply:
yes, you are right, i suggested you because of prd